docs/Developers/Git_Basics.rst

```diff
@@ -453,3 +453,9 @@ use an interactive rebasing session. This opens an editor where you can enter
 commands (described below) for each commit to be rebased. These commands
 determine how individual commits will be transferred to the new base. You can
 also reorder the commit listing to change the order of the commits themselves.
+
+If you would like more information/commands, please use your favourite search
+engine to look for it. You could also read the
+`git documentation <https://git-scm.com/doc>`_ to learn more. Git is widely used
```

Comment:
I disagree. They would still need to search for commands even if they have this doc. It is not a cake walk once you visit the website. The point of providing the link is to suggest that it is one of the most authentic places to look for git commands.